unitex/src/fr/umlv/unitex/config/Config.java

```diff
@@ -71,6 +76,13 @@
  * @author Sébastien Paumier
  */
 public class Config {
+	
+	private static final int PORT = 9999;
```

Comment:
What about to override the default settings (behavior and port number) appending the next two options to [.unitex.cfg](https://github.com/UnitexGramLab/gramlab-ide/blob/master/unitex/src/fr/umlv/unitex/config/Config.java#L625) and [.gramlab](https://github.com/UnitexGramLab/gramlab-ide/blob/master/src/main/java/fr/gramlab/GramlabConfigManager.java#L53)?

```java
allow_only_one_application_instance = true
one_application_instance_port_number = 49627
```